src/kernel/capinv.c

```diff
@@ -644,13 +623,16 @@ cap_thd_op(struct cap_thd *thd_cap, struct thread *thd, struct pt_regs *regs,
 		arcv_cap = (struct cap_arcv *)captbl_lkup(ci->captbl, arcv);
 		if (!CAP_TYPECHK_CORE(arcv_cap, CAP_ARCV)) return -EINVAL;
 
-		rcvt = arcv_cap->thd;
+		rcvt     = arcv_cap->thd;
```

Comment:
But they're split by an if statement, and the upper one isn't aligned with anything else... I think the extra spacing adds nothing here.